vault/resource_mfa_pingid_test.go

```diff
@@ -0,0 +1,53 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestMFAPingIDBasic(t *testing.T) {
+	mfaPingIDPath := acctest.RandomWithPrefix("mfa-pingid")
```

Comment:
Shadow declaration of `mfaPingIDPath()`.

It's preferable to use generic variable names in this scope.
```suggestion
	path := acctest.RandomWithPrefix("mfa-pingid")
```

Reply:
Fixed in 5d506d7